src/components/DpEditableList/DpEditableList.vue

```diff
@@ -24,7 +24,7 @@
             variant="subtle"
             @click.prevent="showUpdateForm(index)" />
           <dp-button
-            class="ml-0.5"
+            class="ml-1"
```

Comment:
as see in the discussion in the thread, the spacing should differ between dark and light modes. Is it right so?

Reply:
@riechedemos as i see in the tread it should be different for the dark/light mode --> https://demosdeutschland.slack.com/archives/C05BV588CS0/p1742555850819769?thread_ts=1741870101.840919&cid=C05BV588CS0